bin/install-qa-check.d/90config-impl-decl

```diff
@@ -0,0 +1,89 @@
+# Check for implicit function declaration warnings in configure logs
+#
+# ebuilds should set the QA_CONFIG_IMPL_DECL_SKIP array to skip known false
+# positives.
+#
+# Some examples of logs to look for:
+# bash: work/bash-5.1/config.log
+#       ^---  easy
+# python: work/Python-3.10.9/config.log
+#         ^---  easy
+# gcc: work/build/config.log
+#      ^---  can be out-of-tree
+# clang: work/x/y/clang-abi_x86_64.amd64/CMakeFiles/CMakeError.log
+#        ^---  can be non-autotools (and very deep)
+# systemd-utils: work/systemd-stable-251.10-abi_x86_64.amd64/meson-logs/meson-log.txt
+#                ^---  can be non-autotools
+#
+# Adapted from macports portconfigure.tcl with love.
+#
+# See also: bug 892651
+
+config_impl_decl_check() {
+	local log_targets=(
+		'config.log'
+		'CMakeError.log'
+		'meson-log.txt'
+	)
+	local find_args=()
+	local logs=()
+	local files=()
+	local lines=()
+	local funcs=()
+	local l
+
+	# Find config logs. Assume the dirs can have spaces in them, even though
+	# that is hella evil and goes against good filesystem manners!
+	for l in "${log_targets[@]}"; do
+		find_args+=( '-name' "${l}" '-o' )
+	done
+	unset -v find_args[-1]
+	readarray -d '' -t logs < <(printf '%s\0' "${WORKDIR}" | find -files0-from - -type f \( "${find_args[@]}" \) -print0)
```

Comment:
Personally, I would split this line into two

```suggestion
	readarray -d '' -t logs < <(printf '%s\0' "${WORKDIR}" |\
	    find -files0-from - -type f \( "${find_args[@]}" \) -print0)
```

Reply:
Sure. I saw some other QA checks with long lines so I wasn't sure what the recommended style on that was.